api/app.js

```diff
@@ -1,21 +1,26 @@
 const express = require('express');
 const app = express();
 const bodyParser = require('body-parser');
- 
-const mongoose = require('./db/mongoose');
-
+require('dotenv').config();
 
+const port = process.env.PORT || 3000;
 /* ---Load Mongoose Models--*/
-const { List, Task, User } = require('./db/models');
-const jwt = require('jsonwebtoken');
+const {
+    List,
+    Task,
+    User
+} = require('./db/models');
+const {
+    verify
```

Comment:
Instead of having functions that take a callback function, I promisifed the function to avoid callback hell.